### PR TITLE
west/ezflashcli: add tool_opt capability

### PR DESCRIPTION
**Pull request by ydamigos**
Add tool_opt capability

### CODE REVIEW

File: scripts/west_commands/runners/ezflashcli.py [hunk 1]
```diff
@@ -3,43 +3,49 @@
...
         return 'ezflashcli'
 
     @classmethod
     def capabilities(cls):
-        return RunnerCaps(commands={'flash'}, erase=True, reset=True)
+        return RunnerCaps(commands={'flash'}, tool_opt=True, erase=True, reset=True)
+
+    @classmethod
+    def tool_opt_help(cls) -> str:
+        return "Additional options for ezFlashCLI e.g. '-j <serial_number>'"
```
Comment (thedjnK) [inline scripts/west_commands/runners/ezflashcli.py:37]: This doesn't look right, there is a dedicated device ID in `scripts/west_commands/runners/core.py`:
```
    - dev_id: whether the runner supports device identifiers, in the form of an
      -i, --dev-id option. This is useful when the user has multiple debuggers
      connected to a single computer, in order to select which one will be used
      with the command provided.
```
This should be using that

**Review by thedjnK — CHANGES_REQUESTED** [review]

File: scripts/west_commands/runners/ezflashcli.py (hunk 1 above)
Comment (ydamigos) [reply scripts/west_commands/runners/ezflashcli.py]: Added dev_id capability.

**Review by thedjnK — APPROVED** [review]

**Review by blauret — APPROVED** [review]

**Comment by blauret** [comment]
@mbolivar-ampere ping

**Comment by henrikbrixandersen** [comment]
Assigning @blauret as well as this runner is for Renesas SmartBond platforms.

**Review by henrikbrixandersen — APPROVED** [review]

**Pull request merged**